### ISSUE TITLE
Another quirk in docs. (Sorry.)

### ISSUE THREAD

**Issue by MichaelTiernan** [michaelrsweet/epm#76]
I feel like I'm complaining. Sorry. I'm only offering feedback to try to be helpful.
(You give us your work, we should try to help when we can.)
___
I checked out the current version, did a configure, went to the docs directory and did a `make clean` and the makefile deleted:

~~~sh
$ make clean
/bin/rm -f epm-book.epub epm-book.html epm-book.pdf epm.html epminstall.html mkepmlist.html setup.html mantohtml mantohtml.o
~~~

It seems odd that the entries for `epm-book.epub` `epm-book.html` `epm-book.pdf` `epm.html` `epminstall.html` `mkepmlist.html` which are provided as parts of the distribution should be deleted in a clean sweep. It seems like they should either not be part of the distribution and rebuilt on demand and thereby cleaned or should remain untouched and included with a different name to keep them "safe".

Again, thanks for your work!

**Comment by michaelrsweet** [comment]
@MichaelTiernan no worries, happy to have the feedback and give EPM a little love...

**Comment by michaelrsweet** [comment]
I agree these should not be deleted, fixing that now...

**Comment by michaelrsweet** [comment]
[master 6eabf6d] Fix some build system issues (Issue #75, Issue #76)
